### PR TITLE
Add automated accessibility tests for some pages

### PR DESCRIPTION
**Pull request by callumacrae**

### CODE REVIEW

File: app/views/users/sessions/login_email_sent.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h2 class="govuk-heading-l">Check your email</h2>
+    <h1 class="govuk-heading-l">Check your email</h1>
```
Comment (kicferk1) [inline app/views/users/sessions/login_email_sent.html.erb:3]: I always wanted to sort out our header mess, potentially there is still some, but slightly less now :)

File: spec/cypress/integration/a11y.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+describe("Accessibility", () => {
```
Comment (seb-siem-softwire) [inline spec/cypress/integration/a11y.js:1]: At some point, I would recommend sitting down and exploring page object pattern architecture or those tests will become unmaintainable in the future. Good for now but in months time lets have a chat about improving the architecture

**Review by seb-siem-softwire — APPROVED** [review]

**Review by kicferk1 — APPROVED** [review]

File: spec/cypress/integration/a11y.js (hunk 2 above)
Comment (callumacrae) [reply spec/cypress/integration/a11y.js:1]: There are architectural problems with having these tests separate to the behavioural tests, but I don't see how using the page object pattern would fix them in this context.

Remember that these are functional tests, not unit tests - functionality shouldn't be split across multiple tests, so the way we're using `.as()` with Cypress already solves the problem of selectors changing.

Also, using `data-cy` or `data-test` attributes as the Cypress best practices suggests seems a much nicer approach than maintaining a library of selectors in our tests.

Comment (kicferk1) [reply spec/cypress/integration/a11y.js:1]: Cypress best practices can suggest what they want, I don't really like the idea of having a bunch of attributes stumbling around, with the sole purpose of making my test architecture easier.

Key reason being, when I read the html and try to understand what it's doing and why, the last thing I want to be thinking about it 'ah, nice attribute, it makes our tests less brittle'.

I think I'd rather maintain the selectors :P

Comment (callumacrae) [reply spec/cypress/integration/a11y.js:1]: There's rails plugins to remove attributes when they're sent to actual users. We have attributes for our CSS and JavaScript in the HTML, why is attributes to stop us having to maintain a (brittle) selector library any worse?

**Pull request merged**